cinema/models.py

```diff
@@ -94,7 +94,9 @@ def validate_ticket(row, seat, cinema_hall, error_to_raise):
             if not (1 <= ticket_attr_value <= count_attrs):
                 raise error_to_raise(
                     {
-                        ticket_attr_name: f"{ticket_attr_name} number must be in available range: "
+                        ticket_attr_name:
+                        f"{ticket_attr_name} "
+                        f"number must be in available range: "
```

Comment:
redundant f-string

Reply:
No, it is not. I just reformatted code that was not written by me with black:)